pkg/apis/autoscaling/v1alpha1/kpa_types.go

```diff
@@ -73,9 +73,18 @@ type PodAutoscalerSpec struct {
 
 	// ConcurrencyModel specifies the desired concurrency model
 	// (Single or Multi) for the scale target. Defaults to Multi.
+	// Deprecated in favor of ContainerConcurrency.
```

Comment:
If this is Deprecated, that suggests that it still works... it looks like we've removed everything but the external switch. I'd say this is "Discontinued" in that case, as this doesn't control anything.